### PR TITLE
Added Sample app for SliverAppBar Widget

### PR DESCRIPTION
**Pull request by TheGuyDangerous**
**What does this implement/fix? Explain your changes**
Added a sample app/folder for showcasing SliverAppBar Widget.

**Does this close any currently open issues?**
- [ ] Yes
- [x ] No

**Any relevant logs, error output, bug-report etc?**

**Any other comments?**

+ **Where has this been tested?**
Android Emulator and Android 12 physical device

+ **Target Platform:**
Android/IOS

+ **Configuration Information:**

+ **Misc:**

### CODE REVIEW

File: sliver_app_bar_example/android/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        jvmTarget = '1.8'
+    }
+
+    sourceSets {
+        main.java.srcDirs += 'src/main/kotlin'
+    }
+
+    defaultConfig {
+        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
+        applicationId "com.example.sliverappbar.sliver_app_bar_example"
```
Comment (nisrulz) [inline sliver_app_bar_example/android/app/build.gradle:47]: From readme, PR instructions.

Screenshot 2022-10-03 at 15 44 06

File: sliver_app_bar_example/test/widget_test.dart [hunk 2]
```diff
@@ -0,0 +1,30 @@
+// This is a basic Flutter widget test.
```
Comment (nisrulz) [inline sliver_app_bar_example/test/widget_test.dart:1]: If you are not going to add a test, then delete this test file.

File: sliver_app_bar_example/lib/main.dart [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      title: 'Flutter Demo',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: const MyHomePage(),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
```
Comment (nisrulz) [inline sliver_app_bar_example/lib/main.dart:23]: Extract this class file into its own class.

Comment (TheGuyDangerous) [reply sliver_app_bar_example/lib/main.dart:23]: Hi , thanks for your valuable review to my code , 
I've made all the necessary changes requested by you , although confused about this one here .
Could you please elaborate what you meant by "Extract this class file into its own class file" , as I am a beginner at flutter for now and can't understand heavy technical references.

**Comment by TheGuyDangerous** [comment]
Hi @nisrulz , thanks a lot for the code review!
I've updated the code according to the changes requested.

File: sliver_app_bar_example/android/app/build.gradle [hunk 4]
```diff
@@ -44,7 +44,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.example.sliverappbar.sliver_app_bar_example"
+        applicationId "github.nisrulz.sliver_app_bar_example"
```
Comment (nisrulz) [inline sliver_app_bar_example/android/app/build.gradle:47]: The package name doesn't just change in one place. I only highlighted one instance. You will have to do a refactoring task to change it everywhere.

**Comment by nisrulz** [comment]
Just one comment and this should be good to merge once that is resolved.

File: sliver_app_bar_example/android/app/src/main/AndroidManifest.xml [hunk 5]
```diff
@@ -0,0 +1,34 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.sliverappbar.sliver_app_bar_example">
```
Comment (nisrulz) [inline sliver_app_bar_example/android/app/src/main/AndroidManifest.xml:2]: Here is another instance.

File: sliver_app_bar_example/android/app/build.gradle (hunk 4 above)
Comment (TheGuyDangerous) [reply sliver_app_bar_example/android/app/build.gradle:47]: Ya , thanks for the review again , and sorry for the silly mistakes :) 
Updated the package name again , should be good to merge.

**Comment by TheGuyDangerous** [comment]
Hi , eagerly waiting for you to merge the Pr :). It's been a long time since you reacted to my comment.

**Pull request merged**